First draft of the chapter 2 (#8)
sgibson91/cross-stitch-carpentry: _episodes/02-getting-started.md

---
title: "Getting Started"
teaching: 20
exercises: 0
questions:
- "How to find the middle of the fabric?"
- "How to use emroidery hoop?"
- "How to prepare embroidery thread for use?"
objectives:

> - "Gain familiarity with embroidery materials"

Reviewer: ```suggestion
```

- "Getting prepared to stitch"
keypoints:
- ""
---

It is always a good idea to start by looking for the the middle/center of your fabric. 
This will allow you make sure your design is placed as intended in the fabric.

To use the design and materials provided for this lesson, follow these steps:

## Fabric and needle

> - We’re using 14 count aida fabric. This means that 14 stitches will fit in 1 inch of fabric. This type of fabric has well defined holes woven into it making it ideal for cross stitch!

Reviewer: ```suggestion
```

I think I will move this point to prerequisites as I begin to mention it there

>     - The recommended way to do this is to fold the fabric in half and then half again. Another way is to put the needle into the fold to keep track of the centre.

Reviewer: ```suggestion
    - One way to do this is to fold the fabric in half and then half again.
      You can then keep track of the centre by pushing the needle through the fold..
```

    - Here is video for reference: https://youtu.be/nVawkpj1T3g 
   
## Embroidery hoop

- We have provided an embroidery hoop. This will help keep your fabric taut and your tension even while you stitch.

>     - To fit your embroidery hoop: unscrew it and separate the rings, then sandwich the fabric between the inside and outside rings. On a flat surface, push the rings back together trying to keep the centre of the fabric in the centre of the hoop. Gently pull at the loose fabric around the edge while tightening the screw. The fabric should be drum-tight. If the screw is stiff, a flat-head screwdriver will help.

Reviewer: ```suggestion
    - To fit your embroidery hoop: unscrew the fastening and separate the rings, then sandwich the fabric between the inner and outer rings.
      On a flat surface, push the rings back together, trying to keep the centre of the fabric in the centre of the hoop.
      Gently pull at the loose fabric around the edge while tightening the screw.
      The fabric should be drum-tight.
      If the screw is stiff, a flat-head screwdriver will help.
```

   - Here is video for reference: https://youtu.be/4Q4ieJCFBMo 
    
## Thread

> - Now for the thread! Stranded-cotton is the go-to for cross stitch. It’s made of 6 strands and is wound together in skeins. We have provided pre-cut lengths of thread for you to work with. Choose the colour of thread that corresponds to the first stitch in the centre of the pattern. We’re going to work from the centre out.

Reviewer: ```suggestion
- Now for the thread!
  Stranded-cotton is the go-to for cross stitch.
  It’s made of 6 strands and is wound together in skeins.
  First we need to cut a length of thread.
  Choose the colour of thread that corresponds to the first stitch in the centre of the pattern as we’re going to work from the centre out.
```
